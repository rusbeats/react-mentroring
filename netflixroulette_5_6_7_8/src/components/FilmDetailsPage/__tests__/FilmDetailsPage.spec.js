import React from 'react';
import { shallow } from 'enzyme';
import { FilmDetailsPage } from '../index';

const PROPS = {
    match: {
        params: {
            id: 1
        }
    }
};

describe('FilmDetailsPage', () => {
    it('should render correctly (snapshot)', () => {
        const component = shallow(<FilmDetailsPage match={PROPS.match} />);
        expect(component).toMatchSnapshot();
    });
});
