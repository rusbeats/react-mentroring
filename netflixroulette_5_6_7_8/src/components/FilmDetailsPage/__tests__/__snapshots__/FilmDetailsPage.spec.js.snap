// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmDetailsPage should render correctly (snapshot) 1`] = `
<Fragment>
  <Connect(FilmDetails) />
  <Connect(FilmsByGenre) />
  <Footer />
</Fragment>
`;
